recipes/awkward-pandas/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dask-awkward" %}
+{% set version = "2022.9a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/dask_awkward/dask_awkward-{{ version }}.tar.gz
+  sha256: 3cdfbed107813c94bcdd645d05e48160014db406d8d3b75d13a694a06ecd2963
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - hatchling >=1.8.0
+    - hatch-vcs
+    - toml
+  run:
+    - python >=3.8
+    - awkward >=1.10.1
+    - dask
```

Any particular reason this doesn't match upstream with:
```suggestion
    - dask >=2022.01.0
```
?

---

I meant to go back and check when Dask dropped Python 3.7 add that version (happens to be `dask >=2022.02.1`). Going to add this in both places.